tests/integration/BaseTest.cs

```diff
@@ -21,17 +21,17 @@ public BaseTest()
 
         private static void TryLoadFromDotEnv(string fileName)
         {
-            try
+            var path = Path.Combine(Directory.GetParent(Environment.CurrentDirectory).Parent.Parent.FullName, fileName);
+            if (File.Exists(path))
             {
-                var path = Path.Combine(Directory.GetParent(Environment.CurrentDirectory).Parent.Parent.FullName, fileName);
                 DotNetEnv.Env.Load(path, new DotNetEnv.LoadOptions(
                     setEnvVars: true,
                     clobberExistingVars: true,
                     onlyExactPath: true
                 ));
                 System.Diagnostics.Trace.TraceWarning($"{fileName} found. {fileName} file should only be used in local developer computers.");
             }
-            catch
+            else
             {
                 System.Diagnostics.Trace.WriteLine($"{fileName} not found.");
```

Comment:
As the method is void, I guess the rest of the execution is not affected if the else branch is executed. Isn't it better to return bool to not execute the rest of the code?